### ISSUE TITLE
Best way to support aria-labelledby associations for RadioButtonGroup

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/sun#567]
From a thread in https://github.com/phetsims/ohms-law/issues/156.

`aria-labelledby` is used to signify to a screen reader that a group of content is labelledby something else. When cursor/focus first enters the group, the label will be read to the user.

In https://github.com/phetsims/ohms-law/issues/156 it was determined that some of our radio button groups need this association and others don't. Further discussion in the issue is about how to best support this.

@jessegreenberg in https://github.com/phetsims/ohms-law/issues/156#issuecomment-587235010:
>To clarify, I was wondering about using aria-labelledby on every AquaRadioButtonGroup, like we do for RadioButtonGroup so the markup and attributes for all PhET radio buttons is the same. But it sounds like there are important semantic differences between the two and so they should be different.

@zepumph in https://github.com/phetsims/ohms-law/issues/156#issuecomment-587604236
>If we decide to do this, let's think about how it will be implemented.
Here I have the following structure (similarly found in ohms-law and GFL)
```html
<h3>Force Values</h3>

<ul>
  <li>Shown</li>
  <li>Hidden</li>
</ul>
```
>With the following scenery code:
var heading = new Node({tagName:'h3', innerContent:'Force Values'});
var radioButtonGroup = new VerticalAquaRadioButtonGroup( .... );
How would you recommend automatically embedding a general aria-labelledby relationship between an "outside" Node like this heading and the ul of the radiobutton group.

>Right now we would add a third line like:
```
var heading = new Node({tagName:'h3', innerContent:'Force Values'});
var radioButtonGroup = new VerticalAquaRadioButtonGroup( .... );
radioButtonGroup.addArialLabelledbyAssociation({
  thisElementName: AccessiblePeer.PRIMARY_SIBLING,
  otherElementName: AccessiblePeer.PRIMARY_SIBLING,
  otherNode: heading
```
>In the "general" pattern though, there is no way to know otherElementName short of passing it in.
Because of this constraint, I don't currently think that a factored out association would benefit us greatly.

**Comment by jessegreenberg** [comment]
Can we assume there there will never be an "outside node" for the aria-labelledby association? In OL and GFL the label is just on the RadioButtonGroup itself and there isn't an outside node. Maybe the option should be `includeAriaLabelledbyAssociation` (or something) which conditionally adds 

```js
      this.addAriaLabelledbyAssociation( {
        thisElementName: AccessiblePeer.PRIMARY_SIBLING,
        otherNode: this,
        otherElementName: AccessiblePeer.LABEL_SIBLING
      } );
```

**Comment by terracoda** [comment]
@jessegreenberg, is there anything functionally different between the radioButtonGroup and the AquaRadiobuttonGroup or the VerticalAquaRadioButtonGroup. From the names it seems there may only be visual differences.

**Comment by jessegreenberg** [comment]
No functional differences, the only differences are visual.

**Comment by zepumph** [comment]
@jessegreenberg I don't yet see how this fits into a general case. Would you like to discuss at an upcomming meeting?

**Comment by jessegreenberg** [comment]
Discussed today - since RadioButtonGroup.js has the aria-labelledby association we would like to add the aria-labelledby association to VerticalAquaRadioButtonGroup.js as well. At some point, RadioButtonGroup and VerticalAquaRadioButtonGroup will be re-written to consolidate, and at that time we want it to be clear that these two types have the same a11y behavior regardless of their visual representation.

Assigning to me to add to VerticalAquaRadioButtonGroup, remove the custom associations from 
```
inverse-square-law-common/js/view/ISLCForceValuesDisplayControl.js
ohms-law/js/ohms-law/view/UnitsRadioButtonContainer.js
sun/js/demo/ButtonsScreenView.js
```

and check on radio buttons in BASE to make sure they are still OK.

**Comment by jessegreenberg** [comment]
Done in the above commits. The radio button group in BASE looks great. I also verified with one more search that remaining usages of `ariaLabelledbyAssociation` are for things other than radio button groups. Closing.
